src/components/ui/ListEntry.js

```diff
@@ -8,7 +8,7 @@ const Icons = styled.div`
   align-items: center;
   max-height: 36px;
   & > * {
-    padding: 8px;
+    margin: 0 4px;
```

Comment:
We can try adding a selector like `&:not(:last-child)` to add a different margin to the last element here

Reply:
worked amazingly 😍